src/Nebulas/Signer.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Signer.h"
+#include "../HexCoding.h"
+
+using namespace TW;
+using namespace TW::Nebulas;
+
+const char* Signer::TxPayloadBinaryType = "binary";
+const char* Signer::TxPayloadDeployType = "deploy";
+const char* Signer::TxPayloadCallType = "call";
+
+void Signer::sign(const PrivateKey& privateKey, Transaction& transaction) const noexcept {
+    auto hash = this->hash(transaction);
+    transaction.alg = 1;
+    transaction.sign = privateKey.sign(hash, TWCurveSECP256k1);
+}
+
+void Signer::appendBigEndian(Data& data, const uint256_t& value,uint32_t digit) {
```

Comment:
can you tweak and move this function to `BinaryCoding.h`?

Reply:
Ok, but I think it should move to uint256.h. It is more like the **store()** function.